src/components/NoResults.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
```

Comment:
Why was this added, was it to complement NoResultsMessage.js somehow?

Reply:
No clue on this one.. I didn't touch that file..

Reply:
It was probably left over from the last time you worked on the project, but it needs to be removed from the PR so that it is not added when merged